browser-test/src/applicant/application_address_correction.test.ts

```diff
@@ -116,6 +117,24 @@ test.describe('address correction', () => {
     await adminPrograms.gotoAdminProgramsPage()
     await adminPrograms.publishProgram(singleBlockSingleAddressProgram)
 
+    // Create optional address program
+    await adminPrograms.addProgram(optionalAddressProgram)
+
+    await adminPrograms.editProgramBlockWithOptional(
+      optionalAddressProgram,
+      'first block',
+      [],
+      addressWithCorrectionQuestionId,
+    )
+
+    await adminPrograms.goToBlockInProgram(optionalAddressProgram, 'Screen 1')
+    await adminPrograms.clickAddressCorrectionToggleByName(
+      addressWithCorrectionQuestionId,
+    )
+
+    await adminPrograms.gotoAdminProgramsPage()
+    await adminPrograms.publishProgram(optionalAddressProgram)
+
```

Comment:
(blocking) Wrap this with `test.step` and remove the comment.  See example [here](https://github.com/civiform/civiform/wiki/Playwright-Feature-Migration-Guide#steps) and Playwright docs [here](https://playwright.dev/docs/api/class-test#test-step). If you can to through just this `test.beforeAll` that this is in and add `test.step` to each group even better.

By using `test.step` the Playwright trace viewer and console output will better nest things. It's particularly useful in the trace viewer when you need to drill down into all the things being run.

Reply:
Done thanks!

Did the existing sections in https://github.com/civiform/civiform/pull/7221